docs/installation/autoware/docker-installation.md

```diff
@@ -8,6 +8,8 @@
 
 - [Git](https://git-scm.com/)
 
+- For NVIDIA Jetson devices, please install [JetPack](https://docs.nvidia.com/jetson/jetpack/install-jetpack/index.html#how-to-install-jetpack) >= 5.0
```

Comment:
https://developers.google.com/style/word-list?hl=en

> Don't use please in the normal course of explaining how to use a product, even if you're explaining a difficult task. Also don't use the phrase please note.
Use please only when you're asking for permission or forgiveness—for example, when what you're asking for benefits us, inconveniences a reader, or suggests a potential issue with a product.

```suggestion
- For NVIDIA Jetson devices, install [JetPack](https://docs.nvidia.com/jetson/jetpack/install-jetpack/index.html#how-to-install-jetpack) >= 5.0
```